### PR TITLE
First index arbitrum

### PR DESCRIPTION
**Pull request by jihoonsong**
This PR will add two indices on Arbitrum.

- Arbitrum DeFi index.
- Arbitrum Market Cap index.

### CODE REVIEW

File: constants/networks/Arbitrum.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Network } from "../../test/helper/interfaces/network"
+
+export const Arbitrum = function (): Network {
+  return {
+    name: "arbitrum",
+    // url: `${process.env["ARBITRUM_ARCHIVE_NODE_URL"]}`,
```
Comment (ferencdg) [inline constants/networks/Arbitrum.ts:6]: let me send you the Arbitrum chainstack address we use, those public nodes tend to be not so reliable for testing

File: contracts/index/oracles/OracleArbitrum.sol [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+import { OwnableUpgradeable } from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import { UUPSUpgradeable } from "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+import { ContextUpgradeable } from "@openzeppelin/contracts-upgradeable/utils/ContextUpgradeable.sol";
+
+import { IChainlinkAggregatorV3 } from "../dependencies/IChainlinkAggregatorV3.sol";
+import { IIndexOracle } from "../interfaces/IIndexOracle.sol";
+import { Errors } from "../libraries/Errors.sol";
+
+contract OracleArbitrum is
```
Comment (ferencdg) [inline contracts/index/oracles/OracleArbitrum.sol:12]: small thing, but is there anything making this Oracle 'Arbitrum' specific? If not, then we can just use a more general name like ChainlinkOracleWrapper or something like that.

Comment (jihoonsong) [reply contracts/index/oracles/OracleArbitrum.sol:12]: When we support AMM price, OracleArbitrum will support Uniswap V3 while OracleAvalanche won't.

Comment (ferencdg) [reply contracts/index/oracles/OracleArbitrum.sol:12]: We could also have only one oracle for all chains supporting everything, but during configuration (when we specify which AMMs are supported) we configure them based on the chain. It is not that important, so if you changed the URL, I can approve.

**Review by ferencdg — APPROVED** [review]

**Pull request merged**